scipy/signal/tests/test_ltisys.py

```diff
@@ -640,12 +641,198 @@ def test_complex_input(self):
         step(([], [-1], 1+0j))
 
 
-def test_lti_instantiation():
-    # Test that lti can be instantiated with sequences, scalars.  See PR-225.
-    lti([1], [-1])
-    lti(np.array([]), np.array([-1]), 1)
-    lti([], [-1], 1)
-    lti([1], [-1], 1, 3)
+class TestLti(object):
+    """Test the lti base class"""
+    def test_lti_instantiation(self):
+        """
+        Test that lti can be instantiated with sequences, scalars.
+
+        See PR-225.
+        """
+        # TransferFunction
+        s = lti([1], [-1])
+        assert_(isinstance(s, TransferFunction))
+        assert_(isinstance(s, lti))
+
+        # ZerosPolesGain
+        s = lti(np.array([]), np.array([-1]), 1)
+        assert_(isinstance(s, ZerosPolesGain))
+        assert_(isinstance(s, lti))
+
+        # StateSpace
+        s = lti([], [-1], 1)
+        s = lti([1], [-1], 1, 3)
+        assert_(isinstance(s, StateSpace))
+        assert_(isinstance(s, lti))
+
+
+class TestStateSpace(object):
+    """Test StateSpace class"""
```

Comment:
This docstring is a bit superfluous. For all the docstrings for ``test_`` methods, can you please change those to comments? That keeps the output of ``signal.test(verbose=2)`` sane (with docstrings, those get displayed instead of the test class/method names).